README.md

```diff
@@ -3,123 +3,56 @@
-[CDCS]: https://github.com/oxfordcontrol/CDCS
-[CDD]: https://github.com/cddlib/cddlib
-[Clp]: https://github.com/coin-or/Clp
-[COSMO]: https://github.com/oxfordcontrol/COSMO.jl
-[Couenne]: https://projects.coin-or.org/Couenne
-[CPLEX]: https://www.ibm.com/analytics/cplex-optimizer
-[CSDP]: https://projects.coin-or.org/Csdp/
-[ECOS]: https://github.com/ifa-ethz/ecos
-[FICO Xpress]: https://www.fico.com/en/products/fico-xpress-optimization-suite
-[GLPK]: http://www.gnu.org/software/glpk/
-[Gurobi]: https://www.gurobi.com/
-[Ipopt]: https://github.com/coin-or/Ipopt
-[Juniper]: https://github.com/lanl-ansi/Juniper.jl
-[MOSEK]: https://mosek.com/
-[NLopt]: https://nlopt.readthedocs.io/en/latest/
-[OSQP]: https://osqp.org/
-[ProxSDP]: https://github.com/mariohsouto/ProxSDP.jl
-[SCIP]: https://scip.zib.de/
-[SCS]: https://github.com/cvxgrp/scs
-[SDPA]: http://sdpa.sourceforge.net/
-[SDPT3]: https://blog.nus.edu.sg/mattohkc/softwares/sdpt3/
-[SeDuMi]: http://sedumi.ie.lehigh.edu/
-[Tulip]: https://github.com/ds4dm/Tulip.jl
-[linear programming]: https://en.wikipedia.org/wiki/Linear_programming
-[(mixed) integer programming]: https://en.wikipedia.org/wiki/Integer_programming
-[second-order conic programming]: https://en.wikipedia.org/wiki/Second-order_cone_programming
-[semidefinite programming]: https://en.wikipedia.org/wiki/Semidefinite_programming
-[nonlinear programming]: https://en.wikipedia.org/wiki/Nonlinear_programming
-
-JuMP makes it easy to specify and **solve optimization problems without expert knowledge**, yet at the same time allows experts to implement advanced algorithmic techniques such as exploiting efficient hot-starts in linear programming or using callbacks to interact with branch-and-bound solvers. JuMP is also **fast** - benchmarking has shown that it can create problems at similar speeds to special-purpose commercial tools such as AMPL while maintaining the expressiveness of a generic high-level programming language. JuMP can be easily embedded in complex work flows including simulations and web servers.
-
-Our documentation includes an installation guide, quick-start guide, and reference manual. The **[JuMPTutorials.jl]** repository contains a small but growing collection of contributed examples. Submissions are welcome!
-
-[JuMPTutorials.jl]: https://github.com/jump-dev/JuMPTutorials.jl
-
-**See [NEWS](https://github.com/jump-dev/JuMP.jl/tree/master/NEWS.md) for
-a list of the significant breaking changes in the JuMP 0.19 release.**
-
-**Latest Release**: 0.21.5 (`release-0.21` branch)
-  * [Documentation](https://jump.dev/JuMP.jl/v0.21.5/)
+JuMP is a domain-specific modeling language for [mathematical optimization](https://en.wikipedia.org/wiki/Mathematical_optimization)
+embedded in [Julia](https://julialang.org/). You can find out more about us by 
+visiting [jump.dev](https://jump.dev).
+
+**Latest Release**: 0.21.5 (`release-0.21` branch):
+  * Use this version if you are using JuMP to solve problems.
```

Comment:
The difference between a release and development version is pretty clear. Doesn't need a sentence explaining, if we're looking at reducing the text in the readme.